Werewolf for Telegram/Werewolf Node/Werewolf.cs

```diff
@@ -2781,10 +2790,22 @@ private void LynchCycle()
                         var t = choices.FirstOrDefault(x => x.PlayerRole == IRole.Tanner);
                         if (t != null && t.Votes > 0)
                             AddAchievement(t, AchievementsReworked.SoClose);
+                        var ScapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
+                        if (ScapeGoat != null)
+                        {
+                            SendWithQueue(GetLocaleString("ScapeGoatLynched", lynched.GetName()));
+                            KillPlayer(lynched, KillMthd.Lynch, killers: Players.Where(x => x.Choice == lynched.Id), isNight: false);
+                        }
                     }
                     else // if (lynched.Votes == -2) // No lynch votes at all
                     {
                         SendWithQueue(GetLocaleString("NoLynchVotes"));
+                        var ScapeGoat = Players.FirstOrDefault(x => x.PlayerRole == IRole.ScapeGoat && !x.IsDead);
+                        if (ScapeGoat != null)
+                        {
+                            SendWithQueue(GetLocaleString("ScapeGoatLynched", lynched.GetName()));
+                            KillPlayer(lynched, KillMthd.Lynch, killers: Players.Where(x => x.Choice == lynched.Id), isNight: false);
```

Comment:
At *this* point, lynched is an IPlayer object with no properties except for Votes = -2 (see line 2694). Trying to get the name or kill them won't work.
```suggestion
                            SendWithQueue(GetLocaleString("ScapeGoatLynched", scapeGoat.GetName()));
                            KillPlayer(scapeGoat, KillMthd.Lynch, isNight: false);
```
Who are the killers in this case? Basically noone.
Accessing `lynched.Id`, again, will not work because `lynched` is not an actual player at this point.